frontend/www/js/omegaup/components/problem/FilterTags.test.ts

```diff
@@ -0,0 +1,35 @@
+import { mount } from '@vue/test-utils';
+import expect from 'expect';
+
+import { types } from '../../api_types';
+import T from '../../lang';
+
+import collection_Filter_Tags from './FilterTags.vue';
+
+describe('Filter.vue', () => {
+  it('Should handle empty list of tags', async () => {
+    const wrapper = mount(collection_Filter_Tags, {
+      propsData: {
+        collection: <types.CheckedTag[]>[
+          { tagname: 'problemTagMatrices', checked: false },
+          { tagname: 'problemTagDiophantineEquations', checked: false },
+          { tagname: 'problemTagInputAndOutput', checked: false },
+          { tagname: 'problemTagArrays', checked: false },
+        ],
```

Comment:
Este componente no contiene la propiedad `collection`, y tampoco existe el tipo que le asignaste. Así que se puede sustituir por:
```suggestion
        tags: <string[]>[
          'problemTagMatrices',
          'problemTagDiophantineEquations',
          'problemTagInputAndOutput',
          'problemTagArrays',
        ],
```
?